Automattic/wp-calypso
# Pressable: show proper my-plan page

This is a part of #8836.

- Introduces `state.sites.ID.options.pressable` option in sites redux tree
- Introduces `isPressableSite` selector
- Introduces `PRESSABLE_TRANSFER_SUCCESS` action to test pressable internals
- Disables domains manage link in my-plan page

## Testing

- Go to my-plan page for a business site
- Open redux dispatcher from redux devtools
- Dispatch `{ type: 'PRESSABLE_TRANSFER_SUCCESS', siteId: ID }`
- See that button disappears for managing domain OR whole call to action disappears for using domain credit

CC @rralian @gwwar @lamosty @retrofox @adambbecker
